kroxylicious-filters/kroxylicious-record-validation/pom.xml

```diff
@@ -111,7 +111,7 @@
         <dependency>
             <groupId>com.github.spotbugs</groupId>
             <artifactId>spotbugs-annotations</artifactId>
-            <scope>compile</scope>
+            <scope>provided</scope>
```

Comment:
If the scope is set in the dependency management section to provided would we be better removing scope here?

Reply:
done, I've removed all the redundant overrides